soft-delete/packages.config

```diff
@@ -1,15 +1,25 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.KeyVault.Core" version="2.0.4" targetFramework="net462" />
+  <package id="Azure.Storage.Blobs" version="12.10.0" targetFramework="net462" />
+  <package id="Microsoft.Bcl.AsyncInterfaces" version="1.0.0" targetFramework="net462" />
   <package id="Microsoft.Data.Edm" version="5.8.3" targetFramework="net462" />
```

Comment:
Many of these can be deleted, I think - I think this is all we need:

```
  <package id="Azure.Core" version="1.19.0" targetFramework="net462" />
  <package id="Azure.Storage.Blobs" version="12.10.0" targetFramework="net462" />
  <package id="Azure.Storage.Common" version="12.9.0" targetFramework="net462" />
  <package id="Microsoft.Bcl.AsyncInterfaces" version="1.0.0" targetFramework="net462" />
  <package id="System.Buffers" version="4.5.1" targetFramework="net462" />
  <package id="System.Diagnostics.DiagnosticSource" version="4.6.0" targetFramework="net462" />
  <package id="System.Memory" version="4.5.4" targetFramework="net462" />
  <package id="System.Memory.Data" version="1.0.2" targetFramework="net462" />
  <package id="System.Numerics.Vectors" version="4.5.0" targetFramework="net462" />
  <package id="System.Runtime.CompilerServices.Unsafe" version="4.6.0" targetFramework="net462" />
  <package id="System.Text.Encodings.Web" version="4.7.2" targetFramework="net462" />
  <package id="System.Text.Json" version="4.6.0" targetFramework="net462" />
  <package id="System.Threading.Tasks.Extensions" version="4.5.2" targetFramework="net462" />
  <package id="System.ValueTuple" version="4.5.0" targetFramework="net462" />
```

Reply:
Done.